internal/records/record_builder.go

```diff
@@ -20,6 +20,18 @@ const (
 	TagName = "frostdb"
 )
 
+type Record struct {
+	arrow.Record
+	SortingColumns []SortingColumn
+}
+
+type SortingColumn struct {
```

Comment:
This seems to be very close to the `arrowutils.SortingColumn` struct. If we can, I would reuse that here to avoid redefining similar structs in different packages (we might want to extract it?) since long term this creates technical debt and confusion.

Reply:
done 3755706